ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -527,205 +492,418 @@ Control {
-                    visible: count
-                    clip: false
-                    model: root.socialLinks
-                    ScrollBar.vertical: StatusScrollBar { }
-                    delegate: InfoCard {
-                        width: GridView.view.cellWidth - Style.current.padding
-                        height: GridView.view.cellHeight - Style.current.padding
-                        title: model.text
-                        asset.bgColor: Style.current.translucentBlue
-                        asset.name: model.icon
-                        asset.color: ProfileUtils.linkTypeColor(model.linkType)
-                        asset.width: 20
-                        asset.height: 20
-                        asset.bgWidth: 32
-                        asset.bgHeight: 32
-                        asset.isImage: false
-                        subTitle: ProfileUtils.linkTypeToText(model.url, model.linkType)
-                        rightSideButtons: RowLayout {
-                            StatusFlatRoundButton {
-                                implicitWidth: 24
-                                implicitHeight: 24
-                                type: StatusFlatRoundButton.Type.Secondary
-                                icon.name: "external"
-                                icon.width: 16
-                                icon.height: 16
-                                icon.color: Theme.palette.baseColor1
-                                enabled: root.networkConnectionStore.sendBuyBridgeEnabled
-                                onClicked: {
-                                    Global.openSendModal(model.address)
-                                    //TODO https://github.com/status-im/status-desktop/issues/13702
-                                    //on right click open menu
+                width: 279
+                height: 32
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: 20
+                anchors.horizontalCenter: parent.horizontalCenter
+                visible: (webView.count > 0)
+                Rectangle {
+                    anchors.fill: parent
+                    color: Style.current.background
+                    radius: 30
+                    border.color: Theme.palette.baseColor2
+                }
+                Row {
+                    anchors.centerIn: parent
+                    spacing: 4
+                    StatusIcon {
+                        width: 16
+                        height: 16
+                        icon: "info"
+                        color: Theme.palette.directColor1
+                    }
+                    StatusBaseText {
+                        font.pixelSize: 13
+                        text: qsTr("Social handles and links are unverified")
+                    }
+                }
+            }
+        }
+    }
+
+    Component {
+        id: delegatesActionsMenu
+        StatusMenu {
+            id: contextMenu
```

Comment:
We need to compute the implicit width

<img width="379" alt="Screenshot 2024-03-27 at 14 02 31" src="https://github.com/status-im/status-desktop/assets/47811206/2a593126-b082-4ce5-8855-c430bbffcfd9">

Reply:
there is an issue for this https://github.com/status-im/status-desktop/issues/14037